hbase-server/src/main/java/org/apache/hadoop/hbase/backup/HFileArchiver.java

```diff
@@ -569,8 +569,17 @@ private static boolean resolveAndArchiveFile(Path archiveDir, File currentFile,
           + " because it does not exist! Skipping and continuing on.", fnfe);
         success = true;
       } catch (IOException e) {
-        LOG.warn("Failed to archive " + currentFile + " on try #" + i, e);
         success = false;
+        LOG.warn("Failed to archive " + currentFile + " on try #" + i, e);
```

Comment:
Better add some comments about why we need this. People will be confusing that why after the failure of a renaming, we need to delete the target file...